web/static/js/content_types/html_editor.js

```diff
@@ -1,13 +1,13 @@
 import MediumEditor from 'medium-editor'
 
 class HtmlEditor {
-  constructor (thesis) {
-    this.thesis = thesis
```

Comment:
We no longer hang onto an instance of the `ThesisEditor` for `setState` and other reasons. Instead, we pass through callbacks to use `onChange` etc.